bravado/http_future.py

```diff
@@ -30,6 +35,37 @@ def result(self, timeout=None):
         )
 
 
+def raraise_timeout_errors(func):
```

Comment:
r**e**raise_timeout_errors

Reply:
I'm renaming it to a more general ``reraise_errors`` so we could use the same decorator for handling connection errors (not done by this PR)